packages/logging-controller/src/LoggingController.ts

```diff
@@ -107,8 +107,13 @@ export class LoggingController extends BaseController<
    */
   #generateId(): string {
     let id = random();
+    let i = 0;
     while (id in this.state.logs) {
+      if (i > 1000) {
+        throw new Error('Endless loop');
```

Comment:
Do we need to add this change in this PR? It doesn't seem clearly related to `resetMocks`, and it introduces production code changes in a PR that is otherwise focused on tests.

Reply:
I'm guessing that you encountered an infinite loop when updating the mock for `uuid`, and that's what motivated this change

Reply:
Yes, that's correct. True that this doesn't need to be here though, I can remove it.